cscs-checks/libraries/io/netcdf_compile_run.py

```diff
@@ -22,15 +22,23 @@ def __init__(self, lang, linkage):
         self.lang = lang
         self.linkage = linkage
         self.descr = f'{lang_names[lang]} NetCDF {linkage.capitalize()}'
-        self.valid_systems = ['daint:gpu', 'daint:mc', 'dom:gpu', 'dom:mc',
-                              'arolla:cn', 'tsa:cn']
+        self.valid_systems = ['daint:gpu', 'daint:mc',
+                              'dom:gpu', 'dom:mc',
+                              'arolla:cn', 'tsa:cn',
+                              'eiger:mc']
+        if linkage == 'static':
+            self.valid_systems.remove('eiger:mc')
+
```

Comment:
```suggestion
        self.valid_systems = ['daint:gpu', 'daint:mc',
                              'dom:gpu', 'dom:mc',
                              'arolla:cn', 'tsa:cn']
        if linkage == 'dynamic':
            self.valid_systems.append('eiger:mc')

```
Doesn't it make more sense to add it only in case of dynamic instead of removing it?

Reply:
I thought about it, but it's more explicit if you remove it: you express your intent more clearly: static linkage is not supported in `eiger:mc`.